miden/src/cli/compile.rs

```diff
@@ -8,6 +8,9 @@ pub struct CompileCmd {
     /// Path to .masm assembly file
     #[structopt(short = "a", long = "assembly", parse(from_os_str))]
     assembly_file: PathBuf,
+    /// Path to .masl library file
+    #[structopt(short = "l", long = "library", parse(from_os_str))]
```

Comment:
Not for this PR, but would be good to provide the ability to specify multiple library files. Let's create an issue for this (could be a "good first issue").

Reply:
We should probably add also the ability to add all libraries from a directory, maybe recursively.